docs/sources/tempo/traceql/metrics-queries.md

```diff
@@ -94,16 +114,51 @@ To demonstrate this flexibility, consider this nonsensical quantile on `span.htt
 { name = "GET /:endpoint" } | quantile_over_time(span.http.status_code, .99, .9, .5)
 ```
 
-## Exemplars
+### The `compare` function
 
-Exemplars are a powerful feature of TraceQL metrics.
-They allow you to see an exact trace that contributed to a given metric value.
-This is particularly useful when you want to understand why a given metric is high or low.
+This adds a new metrics function `compare` which is used to split the stream of spans into two groups: a selection and a baseline.
+It returns time-series for all attributes found on the spans to highlight the differences between the two groups.
+This is kind of hard to describe so there are some example outputs below:
 
-Exemplars are available in TraceQL metrics for all functions.
-To get exemplars, you need to configure it in the query-frontend with the parameter `query_frontend.metrics.exemplars`,
-or pass a query hint in your query.
+The function is used like other metrics functions: when it's placed after any search query, and converts it into a metrics query:
+`...any spanset pipeline... | compare({subset filters}, <topN>, <start timestamp>, <end timestamp>)`
 
+Example:
+```
+{ resource.service.name="a" && span.http.path="/myapi" } | compare({status=error})
+```
+
+#### Parameters
+
+The `compare` function has three parameters:
+
+1. Required. The first parameter is a spanset filter for choosing the subset of spans. This filter is executed against the incoming spans. If it matches, then the span is considered to be part of the selection. Otherwise, it is part of the baseline.  Common filters are expected to be things like `{status=error}` (what is different about errors?) or `{duration>1s}` (what is different about slow spans?)
+
+2. Optional. The second parameter is the top `N` values to return per attribute. If an attribute exceeds this limit in either the selection group or baseline group, then only the top `N` values (based on frequency) are returned, and an error indicator for the attribute is included output (see below).  Defaults to `10`.
+
+3. Optional. Start and end timestamps in Unix nanoseconds, which can be used to additionally subset the spans in time. These timestamps must both be given, or neither.
```

Comment:
```suggestion
3. Optional. Start and End timestamps in Unix nanoseconds, which can be used to additionally subset the spans in time.  For example the overall query could be looking at the past hour, and the selection window only 5 minute time period in which there was an anomaly. These timestamps must both be given, or neither.
```

Reply:
I updated your change by removing an extra space and adding a missing comma.